### ISSUE TITLE
Enabling cert-manager with letsencrypt

### ISSUE THREAD

**Issue by plexcellmedia** [richdynamix/arc#15]
Hey!

I managed to get HTTPS enabled with paid certificate with these steps:

1. Open port 443 on web service/deployment
2. Create folder tools/secrets/ssl and place certificate and private key there
3. Add to ENV
```
- name: WEB_SSL_FULLCHAIN
  value: ssl/fullchain.pem
- name: WEB_SSL_PRIVKEY
  value: ssl/privkey.key
```
4. Add to Dockerfile
`COPY ./tools/secrets/ssl/ /etc/nginx/ssl/`

But my question is, after installing cert-manager with Helm using tutorial provided here:
https://cert-manager.readthedocs.io/en/latest/getting-started/install.html#installing-with-helm

How can I replace WEB_SSL_FULLCHAIN and WEB_SSL_PRIVKEY to look certificate priovided by letsencrypt?

Has anybody managed to get cert-manager to work with this setup?

**Comment by lucasff** [comment]
I agree, this project seems to be very promising, but without built-in LE support, it's a dealbreaker.

**Comment by richdynamix** [comment]
Happy to accept PR's to include this feature. It's a tricky setup. At present I use this in production but currently use SSL terminated at the load balancer for ease of use.
